basic/http/collect-user.js

```diff
@@ -1,17 +1,26 @@
 const Koa = require('koa');
 const Router = require('koa-router');
+const argon2 = require('argon2');
 
 const app = new Koa(),
  router =  new Router();
 
+ async function hashPassword(str){
+  try {
+    const hash = await argon2.hash(str);
+  } 
+  catch (err) {
+      throw new Error('Error');
+  };
+}
```

Comment:
We don't need to throw this:
```
async function hashPassword(str){
    return argon2.hash(str);
}
```

if you want to make it cleaner we can do this:
```ts
const hashPassword = async(str) = argon2.hash(str);
```

Reply:
RESOLVED 💯